src/components/customRenderers/DateField/DateField.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { withJsonFormsControlProps } from '@jsonforms/react';
+import { ErrorMessage, Label, TextField as ASCTextField } from '@amsterdam/asc-ui';
+import { ControlProps } from '@jsonforms/core';
+import useCustomControl from '../../../hooks/useCustomControl';
+
+const DateField = (props: ControlProps) => {
+	const { data: value = '', path, label, errors } = props;
+	const { isValid, isDirty, isFocused, onFocus, onBlur, onChange } = useCustomControl(props);
```

Comment:
Maken we geen gebruik van een form context?

Reply:
No. JsonForms tracks the data internally (probable with a context).